edgelet/iotedged/src/lib.rs

```diff
@@ -148,9 +150,27 @@ const EDGE_PROVISIONING_BACKUP_FILENAME: &str = "provisioning_backup.json";
 /// This is the name of the settings backup file
 const EDGE_SETTINGS_STATE_FILENAME: &str = "settings_state";
 
+/// This is the name of the hybrid X509-SAS key file
+const EDGE_HYBRID_IDENTITY_MASTER_KEY_FILENAME: &str = "iotedge_hybrid_key";
+/// This is the name of the hybrid X509-SAS initialization vector
+const EDGE_HYBRID_IDENTITY_MASTER_KEY_IV_FILENAME: &str = "iotedge_hybrid_iv";
+/// Size in bytes of the master identity key
```

Comment:
Mention in these comments that the key and IV lengths is chosen to correspond to AES256-GCM since that's what the HSM library uses.